src/main/java/nopol_examples/nopol_example_4/NopolExample.java

```diff
@@ -11,6 +11,7 @@ public boolean canBeDividedby3(String a) {
 		int initializedVariableShouldBeCollected = 2;
 		int uninitializedVariableShouldNotBeCollected;
 		int otherInitializedVariableShouldBeCollected;
+                int foo = 1;
 		
 		if ( a.length() == 0 ){
```

Comment:
This line (16) has been identified with a suspiciousness value of 10.83%.
The following failing tests covered this line: `nopol_examples.nopol_example_4.NopolExampleTest#test10`,`nopol_examples.nopol_example_4.NopolExampleTest#test5`